src/website/src/pages/ec/components/modal/index.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
in the thumbnail i see the buttons (to me this is a particular use case, it shouldn't be the default for a modal component), aligned to the left, but they are right aligned in the implementation, actually it's the flex direction, row-reverse, doing it, but this is not intuitive, users will pass one button after the other (again, to me this button thingy should simply not be there) and be probably suprised to see them reversed in the order...